public/javascripts/controller/data.js

```diff
@@ -275,6 +301,22 @@ angular.module('undp-ghg-v2')
               function(item) {
                 rowEntity.ga_gas = item;
               });
+          } else if (columnDef.field == 'nk_notation_object.nk_name') {
```

Comment:
@gilgad I think it might get a little slow to do this.... I had first loaded the page with the local copies and did a loop to check if it exists in the array.  Let's not burden the server with each potential record being imported.

Reply:
@matjames007 A local copy is being used when the page loads, that it uses for import. However, I agree that the block of code related to lookup needs to be updated. I made a comment at the beginning of the block that each lookup should efficiently use the local copy instead of making a server request which they currently do.

Reply:
ok cool... just drop it in a `$scope.notation_keys` object and use it.  I had initially written code to query the server each time - but I am not sure how it would scale when we are going to import all sector data.